src/libopensc/ctx.c

```diff
@@ -356,34 +356,6 @@ load_parameters(sc_context_t *ctx, scconf_block *block, struct _sc_ctx_options *
 	return err;
 }
 
-static void load_reader_driver_options(sc_context_t *ctx)
-{
-	struct sc_reader_driver *driver = ctx->reader_driver;
```

Comment:
You have now broken the possibility of limiting the data to any of the other reader drivers! This should not be moved, I think.